gui/doc/viselements.json

```diff
@@ -1104,10 +1104,15 @@
         "properties": [
           {
             "name": "render",
-            "default_property": true,
             "type": "dynamic(bool)",
             "default_value": "True",
-            "doc": "If True, this part is visible on the page.<br/>\nIf False, the part is hidden and all its content is not displayed."
+            "doc": "If True, this part is visible on the page.<br/>\nIf False, the part is hidden and its content is not displayed."
+          },
+          {
+            "name": "class_name",
+            "default_property": true,
+            "type": "dynamic(str)",
+            "doc": "List of CSS class names that will be associated with the generated HTML Element.\nThis class names will be added to the default <code>taipy-&lt;element_type&gt;</code>."
```

Comment:
`taipy-<element_type>` is kind of unfortunate here.
We are in the 'part' block, right?